create.go

```diff
@@ -284,3 +292,58 @@ func createPIDFile(pidFilePath string, pid int) error {
 
 	return nil
 }
+
+// copyIfNeeded copies the cpuset.cpus and cpuset.mems from the parent
+// directory to the current directory if the file's contents are 0
+func copyIfNeeded(current, parent string) error {
+	var (
+		err                      error
+		currentCpus, currentMems []byte
+		parentCpus, parentMems   []byte
+	)
+
+	if currentCpus, currentMems, err = getSubsystemSettings(current); err != nil {
+		return err
+	}
+	if parentCpus, parentMems, err = getSubsystemSettings(parent); err != nil {
```

Comment:
Missing blank line before `if...` (this comment also applies in other places in this PR).